src/app/dashboard/train/components/ModelTraining.tsx

```diff
@@ -31,7 +31,7 @@ export default function ModelTraining({ setPage }: IProps) {
       </div>
       <div className="absolute top-0 inset-0 leading-8 pt-8 ">
         <h1 className="text-center text-3xl font-logirentBold">
-          Model Is now Being TraIned
+          Model Is now BeIng TraIned
```

Comment:
not correct

Reply:
What is not correct ?

Reply:
Leave the text like that,  the font family works in a  funny way . Do to get it to display the text in all caps,  you have to write it like that

Reply:
Check it on preview

Reply:
I see they left it like that on figma so I'll revert ot to lower case and also revert it to lower case in every file where I have forcefully made it uppercase before

Reply:
but why is it BeIng

Reply:
??

Reply:
It's the style of the font I belive